### PR TITLE
Updated the new feedback section in the doc

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
Signed-off-by: Dishank Tiwari <dtiwari@progress.com>

## Description

A New feedback section has been added with thumbs up and down buttons.

## Related PRs

- https://github.com/chef/chef-web-docs/pull/3325

## Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [x] All tests pass

### CODE REVIEW

**Comment by kagarmoe** [comment]
The border doesn't go completely around the widget. Is this intentional?

Screen Shot 2021-11-30 at 3 43 08 PM

File: themes/docs-new/layouts/_default/infra_resource.html [hunk 1]
```diff
@@ -750,6 +750,6 @@ <h2 id = "examples">Examples</h2>
         {{ end }}
       </div>
     </div>
-    {{ partialCached "feedback" . }}
```
Comment (IanMadd) [inline themes/docs-new/layouts/_default/infra_resource.html:753]: We should keep this.

File: themes/docs-new/assets/sass/_mixins.scss [hunk 2]
```diff
@@ -1,4 +1,8 @@
 @import 'foundation-sites/scss/util/mixins';
+@import "vendor/fontawesome/fontawesome.scss";
+@import "vendor/fontawesome/solid.scss";
+@import "vendor/fontawesome/icons.scss";
+@import "vendor/fontawesome/icons.scss";
```
Comment (IanMadd) [inline themes/docs-new/assets/sass/_mixins.scss]: I don't think this is necessary. We're already bringing in fontawesome in the _typography file, so these resources should be available without this.

Comment (IanMadd) [reply themes/docs-new/assets/sass/_mixins.scss]: Also this section is for mixins, so it's not the right place to include these anyway.

**Pull request closed without merging**